app/helpers/taxons_helper.rb

```diff
@@ -0,0 +1,2 @@
+module TaxonsHelper
```

Comment:
Is this module intentionally empty?

Reply:
Yes - it's auto-generated when creating new controllers/models